### PR TITLE
fix indentation and render types with square brackets

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
See if you like these changes

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1 [hunk 1]
```diff
@@ -83,9 +85,9 @@ ScriptStackTrace : at <ScriptBlock>, <No file>: line 8
 Exception        :
     Type    : GetErrorPrimitiveArray
     Values  :
-        1
-        2
-        0.5
+              1
```
Comment (jborean93) [inline test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1:88]: This seems wrong it being indented even more than what it was before.

Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1:88]: Here's how it looks:

image

Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1:88]: Values are intended to be indented to align past the colon

Comment (jborean93) [reply test/powershell/Modules/Microsoft.PowerShell.Core/Get-Error.Tests.ps1:88]: Fair enough.

**Pull request merged**